lib/spack/docs/getting_started.rst

```diff
@@ -1119,6 +1119,14 @@ Secret keys may also be later exported using the
       <https://www.digitalocean.com/community/tutorials/how-to-setup-additional-entropy-for-cloud-servers-using-haveged>`_
       provides a good overview of sources of randomness.
 
+Here is an example of creating a key. Note that we provide a name for the key first
+(which we can use to reference the key later) and an email address:
+
+.. code-block:: console
+
+    $ spack gpg create dinosaur dinosaur@thedinosaurthings.com
```

Comment:
Could we consider adding `--secret` to `spack gpg create` as well, if the `--export` flag is on?

Reply:
I don't see why not!

Reply:
For that command, since `--export` is for public, and we only provide one output file, we will only be allowed one or the other.

Reply:
oh scratch that, `--export` isn't a boolean. it's a string that is treated as one. So we can treat it just to store the filename and then do the secret key if `--secret` is provided, otherwise public.

Reply:
Yes! In my attempted change I would use either `--export` or `--export-secret-keys` in the generated gpg command line depending upon the `--secret` flag.

Reply:
I used `--export` and `--export-secret` (it's shorter) - is that ok? https://github.com/spack/spack/pull/22557/commits/9f354a624739ba3d0873192813557378fde13c56

Reply:
I made it a string to match the `--export` argument, so the user can actually export both at once.

Reply:
omg amazing!! I totally didn’t figure that out!!